tests/Aspire.Hosting.Tests/DistributedApplicationTests.cs

```diff
@@ -315,6 +211,8 @@ public async Task TestServicesWithMultipleReplicas()
 
         await app.StartAsync(cts.Token);
 
+        await Task.Delay(1000, cts.Token);
```

Comment:
Why this change? This seems brittle.

Reply:
When I looked at the logs (Console) in the test results it was full of retries from the next httpclient calls (retry policy) because the requests are sent before the application is ready. I added this one just to delay when the requests start to limit how many useless queries are sent, and reduce the log size. It doesn't impact the test itself (IMO).

Reply:
I added a comment